### PR TITLE
more biome

### PR DESCRIPTION
**Pull request by ghost**
added savanna biome

### CODE REVIEW

**Comment by ghost** [comment]
screenshot_2016-05-29-23-08-55
i have more ,but is in testing :+1:

**Comment by robske110** [comment]
Have you registered it properly?

File: src/pocketmine/level/generator/normal/biome/SavannaBiome.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+use pocketmine\level\generator\populator\TallGrass;
+use pocketmine\level\generator\populator\AcaciaTree;
+
+class SavannaBiome extends NormalBiome{
+
+	public function __construct(){
+
+		$trees = new AcaciaTree();
+		$trees->setBaseAmount(4);
+		$this->addPopulator($trees);
```
Comment (inxomnyaa) [inline src/pocketmine/level/generator/normal/biome/SavannaBiome.php:34]: trees and tallGrass are defined? or not.

**Comment by inxomnyaa** [comment]
We'll see..

**Comment by ghost** [comment]
i'm upload 1 per 1 ,not all.

**Comment by Muqsit** [comment]
pocketmine\level\generator\populator\AcaciaTree file doesn' exist..

**Comment by inxomnyaa** [comment]
Do not PR to master. PR to generator.

**Comment by ghost** [comment]
muqsit,in wekeend i'm creating epic generator. :p

**Comment by inxomnyaa** [comment]
(and he doesnt even know how to create pr's :P)

**Comment by Muqsit** [comment]
xD

**Comment by Muqsit** [comment]
Lol btw you might wanna have a look at this old pocketmine issue: https://github.com/PocketMine/PocketMine-MP/issues/1747

**Comment by inxomnyaa** [comment]
> This will be rewritten on the upcoming creation of better world generators.

As if :P

@shoghicp

**Comment by Aericio** [comment]
Lmao 2014 and now its 2016. Good job shoghicp for making a "better" generator in the 2 year time span.

**Pull request closed without merging**